scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -3786,9 +3809,9 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     assert(token.text == "end")
     next()
     if (token.is[Ident]) {
-      EndMarker(termName())
+      EndMarker(autoPos(termName()))
```

Comment:
Should term name not have `autoPos` inside?  There is a number of invocations of `termName()` and most do not seem to be inside `autoPos` or `atPos`.

Reply:
Good point, I removed `autoPos` and the tests still pass so it's not needed